sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleGridComponent.java

```diff
@@ -26,6 +26,8 @@ public abstract class SampleGridComponent<T, C extends BaseCriteria> extends Ver
 
 	private static final long serialVersionUID = 2805771019861077651L;
 
+	public static final String DELETE_REASON_COLUMN = "deleteReasonCumulated";
```

Comment:
this constant is not used in this class, should be deleted or moved to the class(es) where it is used

Reply:
Yes, is not used here, only the child classes are using the same constant:
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/0b79d9a7-7eb1-4027-9f60-baf0ea5cec4c)

Reply:
Those classes already use the constant from `FilteredGrid`, so no need to have another one here